### PR TITLE
[ENTESB-10866] Changes to support new dv image

### PR DESCRIPTION
**Pull request by vhalbert**

### CODE REVIEW

File: templates/fuse-online-image-streams.yml [hunk 1]
```diff
@@ -123,3 +123,21 @@ items:
         importPolicy:
           scheduled: true
         name: "v0.4.7"
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    name: fuse-ignite-komodo
```
Comment (lgarciaaco) [inline templates/fuse-online-image-streams.yml]: Maybe we should stick to the name convention: `syndesis-$name`. Arguably we could change all IS names so `fuse-online-$name`

File: resources/fuse-online-template.yml [hunk 2]
```diff
@@ -1688,3 +1703,132 @@ objects:
...
+      type: Recreate
+    template:
+      metadata:
+        labels:
+          app: syndesis
+          syndesis.io/app: syndesis
+          syndesis.io/type: infrastructure
+          syndesis.io/component: komodo-server
+      spec:
+        serviceAccountName: syndesis-server
```
Comment (lgarciaaco) [inline resources/fuse-online-template.yml]: As discussed in chat, `syndesis-server` isn't necessary here, but I guess as changes are applied

File: resources/fuse-online-template.yml [hunk 3]
```diff
@@ -1688,3 +1703,132 @@ objects:
...
+          configMap:
+            name: syndesis-server-config
+    triggers:
+    - imageChangeParams:
+        automatic: true
+        containerNames:
+        - komodo-server
+        from:
+          kind: ImageStreamTag
+          name: komodo-server:latest
```
Comment (lgarciaaco) [inline resources/fuse-online-template.yml]: I think this should be pinned

File: resources/fuse-online-template.yml (hunk 2 above)
Comment (vhalbert) [reply resources/fuse-online-template.yml]: So on master, this applies, but for backporting  to 1.6.x, this shoudn't be?

File: templates/fuse-online-image-streams.yml (hunk 1 above)
Comment (vhalbert) [reply templates/fuse-online-image-streams.yml]: Most of the names begin with fuse-ignite.  Are you saying this should be changed?

**Pull request merged**